x-pack/solutions/observability/plugins/observability_onboarding/ui_tests/playwright.config.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-import { createPlaywrightConfig } from '@kbn/scout';
+import { createPlaywrightConfig } from '@kbn/scout-oblt';
```

Comment:
Oblt plugins should import only from `@kbn/scout-oblt` as it might override the default behaviour of some fixtures/page objects defined in `@kbn/scout`